xbmc/pvr/epg/EpgSearchData.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (C) 2012-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "EpgSearchData.h"
+
+#include "ServiceBroker.h"
+#include "pvr/PVRManager.h"
+#include "utils/log.h"
+
+using namespace PVR;
+
+void PVREpgSearchData::Reset()
```

Comment:
Is this file used?

Reply:
I can drop it and do local build. But I'd clean it with separate PR not to poison this one.
Same applies to copyright date changes mentioned above